src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP4200/CREATE2_EOF1Filler.yml

```diff
@@ -0,0 +1,249 @@
+      # Initcode: PUSH1(1) RJUMPI(3) NOP NOP STOP PUSH1(20) PUSH1(39) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN
+      - ':label eof1_initcode_rjumpi_pos :raw 0xef00010100040200010014030014000000000360015d00035b5b006014602760003960146000f3ef000101000402000100010300000000000000fe'
+      # - Negative:
+      # Initcode (data): ef000101000402000100010300000000000000fe (code to be deployed)
+      # Initcode: RJUMP(12) PUSH1(20) PUSH1(40) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN PUSH1(1) RJUMPI(-17) STOP
+      - ':label eof1_initcode_rjumpi_neg :raw 0xef0001010004020001001503001400000000035c000c6014602860003960146000f360015dffef00ef000101000402000100010300000000000000fe'
+      # - Zero:
+      # Initcode (data): ef000101000402000100010300000000000000fe (code to be deployed)
+      # Initcode: PUSH1(1) RJUMPI(0) PUSH1(20) PUSH1(36) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN
+      - ':label eof1_initcode_rjumpi_zer :raw 0xef00010100040200010011030014000000000360015d00006014602460003960146000f3ef000101000402000100010300000000000000fe'
+
+      # Valid EOF initcode with RJUMPV table size 1
+      # - Positive:
+      # Initcode (data): ef000101000402000100010300000000000000fe (code to be deployed)
+      # Initcode: PUSH1(0) RJUMPV(3) NOP NOP STOP PUSH1(20) PUSH1(39) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN
+      - ':label eof1_initcode_rjumpv_1_pos :raw 0xef00010100040200010014030014000000000360015d00035b5b006014602760003960146000f3ef000101000402000100010300000000000000fe'
+      # - Negative:
+      # Initcode (data): ef000101000402000100010300000000000000fe (code to be deployed)
+      # Initcode: RJUMP(12) PUSH1(20) PUSH1(41) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN PUSH1(0) RJUMPV(-18) STOP
+      - ':label eof1_initcode_rjumpv_1_neg :raw 0xef0001010004020001001603001400000000035c000c6014602960003960146000f360005e01ffee00ef000101000402000100010300000000000000fe'
+      # - Zero:
+      # Initcode (data): ef000101000402000100010300000000000000fe (code to be deployed)
+      # Initcode: PUSH1(0) RJUMPV(0) PUSH1(20) PUSH1(37) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN
+      - ':label eof1_initcode_rjumpv_1_zer :raw 0xef00010100040200010012030014000000000360005e0100006014602560003960146000f3ef000101000402000100010300000000000000fe'
+
+      # Valid EOF initcode with RJUMPV table size 3
+      # Initcode (data): ef000101000402000100010300000000000000fe (code to be deployed)
+      # Initcode: PUSH1(0) RJUMPV(3, 0, -10) NOP NOP STOP PUSH1(20) PUSH1(44) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN
+      - ':label eof1_initcode_rjumpv_3 :raw 0xef00010100040200010019030014000000000360005e0300030000fff65b5b006014602c60003960146000f3ef000101000402000100010300000000000000fe'
+
+      # Valid EOF initcode with RJUMPV table size 255
+      # Initcode (data): ef000101000402000100010300000000000000fe (code to be deployed)
+      # Initcode: PUSH1(0) RJUMPV(3, 0, -10) NOP NOP STOP PUSH1(20) PUSH1(44) PUSH1(0) CODECOPY PUSH1(20) PUSH1(0) RETURN
```

Comment:
This looks to me like a copy paste error

Reply:
yes, thanks